students/juanmayorgahenao.Rmd

```diff
@@ -33,13 +33,149 @@ Currently, I don't have data related to the specific research question stated ab
 
 ```{r}
 # read csv
-d = read.csv('data/juanmayorgahenao_hunterdata.csv')
-surgeon <- subset(d, Species == "Surgeonfish")
-trout <- subset(d, Species == "Coral Trout")
-spotted <- subset(d, Species == "Spotted Damselfish")
-yellow <- subset(d, Species == "Yellow Damselfish")
+d1 = read.csv('data/juanmayorgahenao_hunterdata.csv')
+surgeon <- subset(d1, Species == "Surgeonfish")
+trout <- subset(d1, Species == "Coral Trout")
+spotted <- subset(d1, Species == "Spotted Damselfish")
+yellow <- subset(d1, Species == "Yellow Damselfish")
 densities <- data.frame(surgeon$Adults, trout$Adults, spotted$Adults, yellow$Adults)
 colnames(densities) <- c("Surgeon", "Coral Trout", "Spotted Damselfish", "Yellow Damselfish")
 # output summary
 summary(densities)
 ```
+
+
+
+## Wrangling data
+
+### Reading Data with readr and dplyr
+```{r, results = 'asis'}
+suppressWarnings(library(readr))
+suppressWarnings(suppressMessages(library(dplyr)))
+d = read_csv('../data/r-ecology/species.csv') %>%
+  tbl_df() 
+
+knitr::kable(head(d))
```

Comment:
yay, `kable` :+1: to zoom ahead, for quick interactive tables, try [`DT::datatable`](https://rstudio.github.io/DT/)